src/name-resolver.ts

```diff
@@ -107,6 +113,9 @@ export class NameResolver extends Logger {
    * @return     Promise, that resolves to {string} address
    */
   public async getContent(name: string) {
+    if (!name) {
```

Comment:
Do we need this check here? This function is using `getAddressOrContent`, where we have another check.